code/patches/react-docgen+6.0.0-alpha.4.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/react-docgen/dist/FileState.js b/node_modules/react-docgen/dist/FileState.js
+index ed5f2de..9dabf8f 100644
+--- a/node_modules/react-docgen/dist/FileState.js
++++ b/node_modules/react-docgen/dist/FileState.js
+@@ -10,8 +10,11 @@ var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (
+     return kind === "m" ? f : kind === "a" ? f.call(receiver) : f ? f.value : state.get(receiver);
+ };
+ var _FileState_importer;
+-import traverse, { NodePath } from '@babel/traverse';
++import * as babelraverse from '@babel/traverse';
```

Comment:
How did this surface as a problem? While executing in Node.js or is react-docgen bundled? If so which bundler?

I tried different things myself with this import and ended on what it is now because babel is released in a way that Node.js is not seeing the default export.